common-items.lic

```diff
@@ -28,6 +28,7 @@ module DRCI
                 /That will be (.*) copper (?:kronar|lirum|dokora)s? please/,
                 /to you for (.*) (?:kronar|lirum|dokora)s?/,
                 /I ask (.*) copper (?:kronar|lirum|dokora)s or if you would prefer/,
+                /Cost you just (.*) (?:kronar|lirum|dokora)s?, okie dokie?/,
```

Comment:
❓ Is this case sensitive?

💡 The `?` at the end should be escaped

Reply:
Looks like I only thought about alterations to the end and never got around to implementing them.  One part to double check is if the "i" setting at the end is supported, or is there a different way to set case insensitive.  Or is it preferred to set the case to match the line?

Reply:
`/case SENSItive/i` is supported in ruby to match case insensitive, looks like you may need to add it, add it as the currency's are capitalized http://rubular.com/r/CIRM4eLjDY

Reply:
stick an `i` in that box to the right of the regular expression and it will match